about.html

```diff
@@ -12,52 +12,87 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+  <div class="container">
+  <header>
+    <nav class= "headerBar">
+        <div id="navLogo">
+            <img src="img/lambda-black.png"/>
+        </div>
+        <a href="index.html" class="navButtons">Home</a>
+        <a href="about.html" class="navButtons">About</a>
+        <a href="#" class="navButtons">Products</a>
+        <a href="#" class="navButtons">Blog</a>
+        <a href="#" class="navButtons">Contact</a>
+    </nav>
+    <div id="header-img">
+        <img src="img/jumbo-about.png"/>
+    </div>
+  </header>
+
+
+
+  <section class="sectionContainer">
+    <div class="sectionContent">
+    <h1>About</h1>
+
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </div>
+  </section>
+
+ <div class="about-content">
     
```

Comment:
Nice use of descriptive class names!  When I reference the CSS file I know exactly which area on the about page the CSS is styling.